build-system/tasks/runtime-test.js

```diff
@@ -22,6 +22,14 @@ var karmaConfig = config.karma;
 var extend = require('util')._extend;
 var fs = require('fs');
 var path = require('path');
+var util = require('gulp-util');
+var webserver = require('gulp-webserver');
+var app = require('../test-server').app;
+
+var host = argv.testResponsesServerHost || 'localhost';
+var port = argv.testResponsesServerPort || 9987;
```

Comment:
You don't really allow overriding this in the test. Lets use something higher and more random like `31862` :)